osquery/tables/system/windows/groups.cpp

```diff
@@ -0,0 +1,142 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#define _WIN32_DCOM
+#define WIN32_LEAN_AND_MEAN
+#include <Windows.h>
+// clang-format off
+#include <LM.h>
+// clang-format on
+
+#include <osquery/core.h>
+#include <osquery/tables.h>
+#include <osquery/logger.h>
+
+#include "osquery/core/process.h"
+#include "osquery/core/windows/wmi.h"
+#include "osquery/core/windows/process_ops.h"
+#include "osquery/tables/system/windows/registry.h"
+#include "osquery/core/conversions.h"
+
+namespace osquery {
+
+// Get the relative identifier (RID) from a security identifier (SID):
+unsigned long getRidFromSid(PSID sidPtr) {

```

Comment:
Considering you're already adding a head for `process_ops.h`, perhaps this could live as a declaration and implementation in `process_ops` for future use?

Reply:
Yes that's a good idea, I'll make that change.